### PR TITLE
Adding stylableButton component

### PR DESCRIPTION
**Pull request by alonshmiel1**

### CODE REVIEW

File: packages/wix-ui-core/src/components/StylableButton/StylableButton.st.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.root {
+  -st-states: disabled;
+  display: inline-flex;
+  align-items: center;
+  position: relative;
+  outline: none;
+  /* height: 36px; */
```
Comment (shlomitc) [inline packages/wix-ui-core/src/components/StylableButton/StylableButton.st.css]: remove comment

File: packages/wix-ui-core/src/components/StylableButton/StylableButtonStyle.st.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  -st-extends: StylableButton;
+  height: value(height);
+  padding: value(padding);
+  border-radius: value(borderRadius);
+  font-family: value(fontFamily);
+  font-size: value(fontSize);
+  line-height: value(lineHeight);
+  font-style: value(fontStyle);
+  font-weight: value(fontWeight);
+  /* padding: value(textDecoration); */
```
Comment (shlomitc) [inline packages/wix-ui-core/src/components/StylableButton/StylableButtonStyle.st.css]: remove

File: packages/wix-ui-core/src/components/StylableButton/index.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  root?: object;
+};
+
+export interface StylableButtonProps {
+  type?: string;
+  disabled?: boolean;
+  onClick?: React.EventHandler<React.MouseEvent<HTMLButtonElement>>;
+  onMouseEnter?: React.EventHandler<React.MouseEvent<HTMLButtonElement>>;
+  onMouseLeave?: React.EventHandler<React.MouseEvent<HTMLButtonElement>>;
+  // styles?: StylableButtonStyles;
```
Comment (shlomitc) [inline packages/wix-ui-core/src/components/StylableButton/index.tsx]: remove

File: packages/wix-ui-core/src/components/StylableButton/StylableButton.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import {stylableButtonDriverFactory} from './StylableButton.driver';
+import {createDriverFactory, isTestkitExists, isEnzymeTestkitExists} from 'wix-ui-test-utils';
```
Comment (shlomitc) [inline packages/wix-ui-core/src/components/StylableButton/StylableButton.spec.tsx]: fix it according to how it's done in `master` now.

Comment (alonshmiel1) [reply packages/wix-ui-core/src/components/StylableButton/StylableButton.spec.tsx]: Thanks!

I tried to change it and ran 'npm test' but got some errors..
lerna ERR! test Errored while running script in 'wix-ui-icons-common'
lerna ERR! execute callback with error
lerna ERR! Error: Command failed: npm run test
lerna ERR! sh: svg2react-icon: command not found

I tried to clone wix-ui again + npm install + npm test, but got the same error..
I will check it tomorrow..

Thank you!

Comment (alonshmiel1) [reply packages/wix-ui-core/src/components/StylableButton/StylableButton.spec.tsx]: In addition,
Do you know how can I make the story to override the default style?

I tried to use commonStyle:
packages/wix-ui-core/stories/StylableButton/index.ts

**Comment by shlomitc** [comment]
merged in #237

**Pull request closed without merging**